File: core/model/src/main/kotlin/com/yingrensheng/core/model/order/Order.kt
package com.yingrensheng.core.model.order

import java.time.Instant

data class Order(
    val orderId: String,
    val title: String,
    val amountLabel: String,
    val exportSpec: String,
    val statusLabel: String,
    val createdAt: Instant,
)

File: server/app/schemas/order.py
from pydantic import BaseModel, ConfigDict, Field


class OrderResponse(BaseModel):
    model_config = ConfigDict(populate_by_name=True, serialize_by_alias=True)

    order_id: str = Field(alias="orderId")
    title: str
    amount_label: str = Field(alias="amountLabel")
    export_spec: str = Field(alias="exportSpec")
    status_label: str = Field(alias="statusLabel")
    created_at: str = Field(alias="createdAt")

